ui/ceval/src/view.ts

```diff
@@ -262,13 +262,28 @@ function getElUci(e: MouseEvent): string | undefined {
   );
 }
 
+function getElPvMoves(e: MouseEvent): (string | null)[]{
+  var pvMoves: (string | null)[] = [];
+
+  $(e.target as HTMLElement)
+    .closest('div.pv')
+    .children()
+    .filter('span.pv-san').each(function() {
+      pvMoves.push($(this).attr("data-board"))
+    })
+
+  return pvMoves;
+}
+
 function checkHover(el: HTMLElement, instance: CevalCtrl): void {
   lichess.requestIdleCallback(
     () => instance.setHovering(getElFen(el), $(el).find('div.pv:hover').attr('data-uci') || undefined),
     500
   );
 }
 
+let pvMoves: string | any[] = [];
```

Comment:
This type doesn't look right? This should probably be `(string | null)[]` as well?

Reply:
Done